ark-testing/src/util.rs

```diff
@@ -0,0 +1,9 @@
+use rand::RngCore;
+
+pub fn random_string() -> String {
+	// Generate a few random bytes and base58 encode them
+	// The entropy should be sufficient to generate unique test-names
+	let mut entropy :[u8; 8] = [0; 8];
+	rand::thread_rng().fill_bytes(&mut entropy);
+	bitcoin::base58::encode(&entropy)
```

Comment:
lol :D worth it to bikeshed and benchmark base58 and bech32 to see which one is faster?